modules/op-deploying-database-for-tekton-hub.adoc

```diff
@@ -0,0 +1,29 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-tekton-hub-with-openshift-pipelines.adoc
+
+:_content-type: PROCEDURE
+[id="deploying-database-for-tekton-hub_{context}"]
+= Deploying database for {tekton-hub}
+
+[discrete]
+.Prerequisites
+
+* Ensure that you are in the `<project_root>/config/` directory, where `<project_root>` is the top level directory of the cloned {tekton-hub} repository.
+
+[discrete]
+.Procedure
+
+. Run the following command:
++
+[source,terminal]
+----
+$ oc apply -f 00-init/ -n tekton-hub
+----
+
+. Wait till the pod is running.
```

Comment:
```suggestion
. When the pod is running, <do something>
```
Or, if it is critical, give them a separate step to check whether the pod is running.

Reply:
Replace multiple instances of "till" (antiquated/colloqial) with the more conventional "until."